tests/perf-test/ReportScreen.perf-test.js

```diff
@@ -125,7 +154,12 @@ function ReportScreenWrapper(args) {
 const runs = CONST.PERFORMANCE_TESTS.RUNS;
 
 test('[ReportScreen] should render ReportScreen with composer interactions', () => {
+    const {triggerTransitionEnd, addListener} = createAddListenerMock();
     const scenario = async () => {
+        await screen.findByTestId('ReportScreen');
+
+        await act(triggerTransitionEnd);
```

Comment:
The rest of the test has comments about what is doing so can you add them here as well, please?